src/python/WMCore/WMSpec/Steps/Executors/DQMUpload.py

```diff
@@ -37,8 +37,10 @@ class DQMUpload(Executor):
     Execute a DQMUpload Step
 
     """
+    def __init__(self):
+        return
```

Comment:
That was the only place there I added. Good to know it now. Thanks Alan, will remove soon